spec/file-icons-spec.coffee

```diff
@@ -21,3 +25,45 @@ describe 'FileIcons', ->
     FileIcons.resetService()
 
     expect(FileIcons.getService()).not.toBe(service)
+
+  
+  describe 'Class handling', ->
```

Comment:
I'm having a hard time getting these specs to work in the `tree-view-spec` file. :( They're failing and I can't figure out why and where. I haven't even changed any of the code.